app/lib/screens/home/home.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:app/navigation.dart';
+import 'package:flutter/material.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Home Page'),
+      ),
+      body: Center(
+          child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text('This is Home Page'),
+          SizedBox(
+            height: 10.0,
+          ),
+          RaisedButton.icon(
```

Comment:
All these new lines hurt my physically, but that's the style :/

Reply:
My main goal for this task is to set up and learn bottomnavbar widget. Editing the design should be easy to do once we finalized our design,